PWGCF/FemtoDream/TableProducer/femtoDreamProducerTask.cxx

```diff
@@ -270,11 +273,31 @@ struct femtoDreamProducerTask {
     Configurable<bool> cfgEvtRCTFlagCheckerLimitAcceptAsBad{"cfgEvtRCTFlagCheckerLimitAcceptAsBad", true, "Evt sel: RCT flag checker treat Limited Acceptance As Bad"};
   } rctCut;
 
+  struct : o2::framework::ConfigurableGroup {
+    Configurable<bool> ConfFlowCalculate{"ConfFlowCalculate", false, "Evt sel: Cumulant of flow"}; // To do
```

Comment:
If you put a prefix you will have the grouped in the hyperloop wagon configuration